frontend/src/components/alert/snackBar.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+import MuiAlert from '@material-ui/lab/Alert';
+import Fade from '@material-ui/core/Fade';
+
+interface snackbarProps {
+  open: boolean;
+  message: string;
+  severity: string;
+  handleClose: undefined;
```

Comment:
Use `() => void` instead of undefined. Example here: https://github.com/HKN-UCSD/hkn-member-portal/blob/modal_refactor_new/src/components/modals/BaseModal.tsx